### PR TITLE
Update v2.1.4 to v2.2.4 with the latest spec v2 changes.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by notlee** [comment]
This failure is not expected, it looks like obt assumes a Sass file exists when building pa11y demos

**Comment by JakeChampion** [comment]
> This failure is not expected, it looks like obt assumes a Sass file exists when building pa11y demos

Nice catch, it is lookig for `_index.scss` when it should look for `main.scss` 👍

**Comment by notlee** [comment]
Ah, it's the import in the demo.scss, not obt

**Comment by notlee** [comment]
We have a readme warning here. I'd maybe expect an error for testing purposes though. Do we throw any readme errors or is it all warnings @Financial-Times/origami-core?

**Comment by JakeChampion** [comment]
> We have a readme warning here. I'd maybe expect an error for testing purposes though. Do we throw any readme errors or is it all warnings @Financial-Times/origami-core?

I believe it is all warnings

**Review by JakeChampion — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ To learn about getting started with other Origami components see the [Origami co
 ## Versions
 
 |version|valid js|valid sass|valid html|valid demos|valid readme|js tests pass|sass tests pass|
-|-------|--------|----------|----------|-----------|------------|-------------|---------------|
+|***---|******|******--|******--|******---|*********|*********-|*********---|
```
Comment (JakeChampion) [inline readme.md:21]: This looks like a find and replace mistake as the table no longer has column headings

**Review by JakeChampion — APPROVED** [review]

**Review by JakeChampion — APPROVED** [review]

**Review by JakeChampion — APPROVED** [review]

**Comment by notlee** [comment]
tagged and released

**Pull request closed without merging**